### DISCUSSION TITLE
Blacklist specific Storybook stories?

### DISCUSSION THREAD

**Discussion by ftzi** [lost-pixel/lost-pixel#387] [Q&A] [answered]
I have one particular flaky storybook story. ~~I might end up masking the whole story so no diffs are generated~~ (apparently there is no way to mask just a specific storybook story), but it would be nice if there is a proper way to ignore specific stories.

Is there currently a way to do it?

Thanks!

**Comment by chriskalmar** [comment, marked as answer]
hey @SrBrahma 

you can disable stories like so:
```
YourStory.parameters = {
    lostpixel: { 
        disable: true 
    },
}
```

and regarding flakyness, you might be interested in one of our posts:
https://www.lost-pixel.com/blog/handling-flaky-visual-regression-tests-with-lost-pixel-platform

**Reply by ftzi** [reply]
Thanks a lot!

I am aware about this flakyness thing! I am already using it. But this specific test, for whatever reason, randomly fails from time to time.

Also, I created this PR some time ago, that further improves the flakyness control:

https://github.com/lost-pixel/lost-pixel/pull/385
